etc/profile-m-z/vlc.profile

```diff
@@ -20,6 +20,14 @@ include disable-passwdmgr.inc
 include disable-programs.inc
 include disable-xdg.inc
 
+mkdir ${HOME}/.cache/vlc
+mkdir ${HOME}/.config/vlc
+mkdir ${HOME}/.local/share/vlc
+whitelist ${HOME}/.cache/vlc
+whitelist ${HOME}/.config/vlc
+whitelist ${HOME}/.local/share/vlc
+whitelist ${MUSIC}
+whitelist ${VIDEOS}
```

Comment:
Refactoring VLC as a whitelisting profile can definately harden it. Most if not all such profiles `include whitelist-common.inc` too (to cover potential UI breakage). Can you please test VLC's functionality with that included before we surprise users?